metadata-transformer/src/main/java/no/unit/nva/metadata/CreatePublicationRequest.java

```diff
@@ -100,9 +116,9 @@ public boolean equals(Object o) {
         }
         CreatePublicationRequest that = (CreatePublicationRequest) o;
         return Objects.equals(getEntityDescription(), that.getEntityDescription())
-               && Objects.equals(getFileSet(), that.getFileSet())
-               && Objects.equals(getContext(), that.getContext())
-               && Objects.equals(getProjects(), that.getProjects())
-               && Objects.equals(getSubjects(), that.getSubjects());
+                && Objects.equals(getFileSet(), that.getFileSet())
```

Comment:
use intellij to auto-generate the hashcode and equals every time you add a new field

Reply:
Fikset